frontend/src/scenes/pipeline/pipelineNodeLogic.tsx

```diff
@@ -79,23 +57,16 @@ export const pipelineNodeLogic = kea<pipelineNodeLogicType>([
                 },
                 {
                     key: [Scene.PipelineNode, id],
-                    name: node.name,
+                    name: '',
```

Comment:
unfortunately we now won't have access to `node.name`, as we're not loading the plugin if someone is looking at metrics tab for example.